relate/templates/base.html

```diff
@@ -1,15 +1,17 @@
 <!DOCTYPE html>
 {% load i18n %}
 {% load static %}
+{% trans relate_site_name as RELATE %}
```

Comment:
Why? I'm not that happy with `RELATE` as a variable name.

Reply:
sorry, it seems I misunderstood you previous comments. Can you revert this commit and modify the previous one? thanks.

Reply:
So, let me guess what you meant.

1. Use `relate_site_name` when the literals IS 'relate-site_name' itself, no need to change to `RELATE`.

1. Avoid translation literals which *contain* `RELATE` in template.

1. For Python literals, avoid use `RELATE` as variable name (if we did that before, correct it).

Am I right?

Reply:
Yep, that sounds right. Sorry for not having been clear and having caused you extra work.

Reply:
NVM.